<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>TranscodeDialog</class>
 <widget class="QDialog" name="TranscodeDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Transcode file</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>390</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title" >
    <string>Thumbnail</string>
   </property>
   <widget class="QLabel" name="thumbnailLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>240</width>
      <height>160</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy>
      <hsizetype>5</hsizetype>
      <vsizetype>0</vsizetype>
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="pixmap" >
     <pixmap resource="pspmovie.qrc" >:/images/sonypsp.png</pixmap>
    </property>
   </widget>
   <widget class="QSlider" name="thumbnailSlider" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>200</y>
      <width>240</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>9999</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance" >
     <bool>false</bool>
    </property>
    <property name="invertedControls" >
     <bool>false</bool>
    </property>
    <property name="tickPosition" >
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval" >
     <number>500</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>80</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Output bitrate</string>
   </property>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>30</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Video:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>70</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Audio:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="VideoBitrateSel" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>20</y>
      <width>69</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>786kbps</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>384kbps</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>256kbps</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="AudioBitrateSel" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>70</y>
      <width>69</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>128kbps</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>96kbps</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>64kbps</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>20</y>
     <width>81</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text" >
    <string>Browse</string>
   </property>
   <property name="icon" >
    <iconset resource="pspmovie.qrc" >:/images/stock_open.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="filenameEdit" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>30</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="pspmovie.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>TranscodeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>378</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>TranscodeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>459</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
